src/main/target/common_pre.h

```diff
@@ -264,6 +264,7 @@
 #define USE_HUFFMAN
 #define USE_PINIO
 #define USE_PINIOBOX
+#define USE_PIN_UP_DOWN
```

Comment:
Instead of adding this here, it is better to add this to the `#if (FLASH_SIZE > 256)` below, as I don't think there are any F3 boards that need this, and flash space on F3 is very limited.